application/app.py

```diff
@@ -334,7 +334,7 @@ def aus_temp_rainfall():
       'year': fire['Year'],
       'avg_annual_temp': fire['Avg Annual Temp'],
       'avg_annual_rainfall' : fire['Avg Annual Rainfall'],
-      'temp_difference': fire['temp_differnce'],
+      'temp_difference': fire['temp_difference'],
```

Comment:
As you suggested, I needed to drop the relevant table in my MongoDB instance and then reprocess everything, but it was quick and once sorted it loaded nicely on the site.